hal/src/main/java/edu/wpi/first/hal/SimDevice.java

```diff
@@ -4,7 +4,17 @@
 
 package edu.wpi.first.hal;
 
-/** A wrapper around a simulator device handle. */
+/**
+ * A wrapper around a simulator device handle.
+ *
+ * <p>Teams: if you are using this class, you are likely confusing it for {@link
+ * edu.wpi.first.wpilibj.simulation.SimDeviceSim}.
+ *
+ * <p>Vendors: This class should be used from inside the device class to define the
+ * properties/fields of the device. Use {@link #create} to get a SimDevice object, then use {@link
+ * #createDouble(String, Direction, double)} or similar to define the device's fields. See {@link
+ * edu.wpi.first.wpilibj.Encoder} for an example implementation.
```

Comment:
Any thoughts about this note?
Teams shouldn't be using it, and the confusion is very plausible, especially with autocomplete/Intellisense.